tests/test_workflow_topology.py

```diff
@@ -0,0 +1,206 @@
+
+    workflow.add_operators([forward_op])
+
+    for i in range(5):
+        workflow.set_input_name(f"Input {i}", forward_op, i)
+        workflow.set_output_name(f"Output {i}", forward_op, i)
+
+    return workflow
+
+
+def workflow_disp_min_max(server_type) -> dpf.core.Workflow:
+    """
+    ┌──────────────┐      ┌───────┐      ┌─────────────┐      ┌─────┐
+    │"data_sources"├─────►│disp_op├─────►│min_max_fc_op├──┬──►│"min"│
+    └──────────────┘      └───────┘      └─────────────┘  │   └─────┘
+                                                          │   ┌─────┐
+                                                          └──►│"max"│
+                                                              └─────┘
+    """
+
+    disp_op = op.result.displacement(server=server_type)
+    min_max_fc_op = op.min_max.min_max_fc(disp_op, server=server_type)
+
+    workflow = dpf.core.Workflow(server=server_type)
+
+    workflow.add_operators([disp_op, min_max_fc_op])
+
+    workflow.set_input_name("data_sources", disp_op.inputs.data_sources)
+    workflow.set_output_name("min", min_max_fc_op.outputs.field_min)
+    workflow.set_output_name("max", min_max_fc_op.outputs.field_max)
+
+    return workflow
+
+
+workflows = {
+    "workflow_forward": workflow_forward,
+    "workflow_forward_5": workflow_forward_5,
+    "workflow_disp_min_max": workflow_disp_min_max,
+}
+workflow_topologies = {
+    "workflow_forward": {
+        "operators": 3,
+        "operator_connections": 1,
+        "data_connections": 1,
+        "exposed_inputs": 1,
+        "exposed_outputs": 2,
+    },
+    "workflow_forward_5": {
+        "operators": 1,
+        "operator_connections": 0,
+        "data_connections": 0,
+        "exposed_inputs": 5,
+        "exposed_outputs": 5,
+    },
+    "workflow_disp_min_max": {
+        "operators": 2,
+        "operator_connections": 1,
+        "data_connections": 0,
+        "exposed_inputs": 1,
+        "exposed_outputs": 2,
+    },
+}
+
+
+@pytest.fixture(
+    params=list(workflows.values()),
+    ids=list(workflows.keys()),
+)
+def workflow(server_type, request) -> dpf.core.Workflow:
+    wf = request.param(server_type)
+    wf.name = list(workflows.keys())[request.param_index]
+    return wf
+
+
+@pytest.fixture()
+def expected_workflow_topology(workflow):
+    return workflow_topologies[workflow.name]
+
+
+@pytest.mark.skipif(
+    not conftest.SERVERS_VERSION_GREATER_THAN_OR_EQUAL_TO_10_0,
+    reason="Operator `workflow_to_workflow_topology` does not exist below 10.0",
+)
+def test_instantiate_workflow_to_workflow_topology_op(server_type):
+    workflow_to_workflow_topology_op = dpf.core.Operator(
+        "workflow_to_workflow_topology", server=server_type
+    )
+
+    assert workflow_to_workflow_topology_op
+
+
+@pytest.mark.skipif(
+    not conftest.SERVERS_VERSION_GREATER_THAN_OR_EQUAL_TO_10_0,
+    reason="Operator `workflow_to_workflow_topology` does not exist below 10.0",
+)
+def test_workflow_get_topology(workflow):
```

Comment:
for this test, could you please replace 
@pytest.mark.skipif(
    not conftest.SERVERS_VERSION_GREATER_THAN_OR_EQUAL_TO_10_0, by the decorator defined in conftest that checks that the error message would be meaningful if a use with a lower version uses get_topology